erizo_controller/erizoClient/src/Room.js

```diff
@@ -45,7 +45,7 @@ const Room = (altIo, altConnectionHelpers, altConnectionManager, specInput) => {
   // Private functions
   const removeStream = (streamInput) => {
     const stream = streamInput;
-    if (stream.stream) {
+    if (stream.stream && !stream.local) {
```

Comment:
Not closing the local stream when unpublishing can be risky in some cases. I'd prefer thinking about how we want to manage this in the API and update it accordingly in another PR